src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -16,7 +20,6 @@ public User login(String login, String password) throws AuthenticationException
 
     private User findByLogin(String login) {
         User user = new User(login, "1234");
```

Comment:
The method 'findByLogin' seems to be returning a hard-coded user. In a real-world scenario, this should be replaced with actual logic to retrieve user details from a database or another storage system.